### PR TITLE
SInfo doesn’t support Android fingerprint out of the box

### PR DESCRIPTION
**Pull request by antonm76**
https://github.com/mCodex/react-native-sensitive-info, see it requires additional efforts to be done (The example in the repo shows how to use this feature and how to build some Android UI based on callbacks.)

### CODE REVIEW

File: mobile/src/auth/refresh-token-storage.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+import { Platform } from 'react-native';
 
 export interface RefreshTokenStorage {
     storeToken(refreshToken: string | null): Promise<void>;
+
     getRefreshToken(): Promise<string>;
 }
 
-export class RefreshTokenFilesystemStorage implements RefreshTokenStorage {
+class RefreshTokenBaseStorage {
```
Comment (santee) [inline mobile/src/auth/refresh-token-storage.ts]: lost 'implements' constraint

File: mobile/src/auth/refresh-token-storage.ts [hunk 2]
```diff
@@ -34,8 +35,33 @@ export class RefreshTokenFilesystemStorage implements RefreshTokenStorage {
                 keychainService: 'ArcadiaAssistant',
                 kSecAccessControl: 'kSecAccessControlTouchIDCurrentSet',
```
Comment (santee) [inline mobile/src/auth/refresh-token-storage.ts]: What happens if it's iOS with IsSensorAvailable = false. So you'd have a combination of touchId=false and this kSecAccessControl.

Comment (antonm76) [reply mobile/src/auth/refresh-token-storage.ts]: Good question. Will check tomorrow.

**Comment by antonm76** [comment]
Closing this pull request; next pull request will be created to merge into 'develop'

**Pull request closed without merging**